client/src/main/java/io/spine/client/TargetBuilder.java

```diff
@@ -78,7 +78,7 @@
  * @param <B>
  *         a type of the builder implementations
  */
-abstract class AbstractTargetBuilder<T extends Message, B extends AbstractTargetBuilder> {
+public abstract class TargetBuilder<T extends Message, B extends TargetBuilder> {
```

Comment:
So why is it public now?

Reply:
Because we reference the class in Javadocs.